Cargo.toml

```diff
@@ -1,5 +1,3 @@
-cargo-features = ["edition2021"]
```

Comment:
This means that users would need at least [Rust 1.56.0](https://github.com/rust-lang/rust/releases/tag/1.56.0) where [edition 2021](https://doc.rust-lang.org/nightly/edition-guide/rust-2021/index.html) becomes stable?

Reply:
we're already in 1.67